.github/workflows/default.yaml

```
@@ -145,13 +159,13 @@ jobs:
           export CPPFLAGS="-I$HOMEBREW_PREFIX/include${CPPFLAGS:+ $CPPFLAGS}"
           export LDFLAGS="-L$HOMEBREW_PREFIX/lib${LDFLAGS:+ $LDFLAGS}"
           export CFLAGS="-Wno-compound-token-split-by-macro${CFLAGS:+ $CFLAGS}" # needed fir ltdl with Xcode
-          ./test-builds.sh ${{ matrix.layer }}
+          ./test-builds.sh ${{ matrix.layer.layer }}
```

```suggestion
          ./test-builds.sh ${{ matrix.layer.name }}
```

BTW, these leftovers exposed a small test-builds.sh bug: That script mishandles an empty string passed as an argument. The current PR code does _not_ pass an empty string to test-builds.sh[^E]. I was going to suggest that we modify this PR line, so that it does pass an empty string if we make a similar mistake in spelling matrix.layer.name, but, after testing, I am _not_ suggesting that because of that test-builds.sh bug that has no trivial fix (that I could find).

[^E]: Instead, test-builds.sh gets no arguments at all (and correctly tests all layers) when called from this current PR code line.